charts/datahub/values.yaml

```diff
@@ -123,6 +123,7 @@ datahub-ingestion-cron:
 elasticsearchSetupJob:
   enabled: true
   image:
+    registry: docker.io
```

Comment:
wouldn't it be better to just set `global.imageRegistry`? This way `image.registry` would be used for overrides.

Reply:
I thought to leave it at the lowest level so every job itself is consistent to use docker.io if someone does not adapt anything. But if one wants to use e.g. a mirror it just needs to be overridden globally.

But you are right, if its set globally it takes precedence over the local config. I may need to adapt the template as well. Which way do you prefer? I can adapt it according to it

Reply:
In the latest commit I adapted the order. What do you think about?